DOCS-5607-jms-publish-consume-example-kt (#1360)
mulesoft/docs-connectors: jms/1.7/modules/ROOT/pages/jms-examples.adoc

Configure the *Consume* operation to consume a message at any given time in the flow, from any given destination.
* xref:jms-publish-consume.adoc[Publish Messages and Listen for Replies] +
Configure the *Publish consume* operation to publish a message to any destination, and then wait for a reply on a different destination.

> * xref:jms-publish-consume-example.adoc[Publish and Consume Messages within Flows] +

Reviewer: ```suggestion
* xref:jms-publish-consume-example.adoc[Publish and Consume Messages Within Flows] +
```

> Configure the *Publish consume* operation to publish and consume messages within two flows, and uses a correlation id to ensure that the incoming messages are the proper reply of the outgoing message.

Reviewer 2: ```suggestion
Configure the *Publish consume* operation to publish and consume messages within two flows, and use a correlation ID to ensure that the incoming messages are the correct reply for the outgoing message.
```

Reviewer: Tweaked:
```suggestion
Configure the *Publish consume* operation to publish and consume messages using two flows and a correlation ID to ensure that incoming messages are replies to a specific outgoing message.
```
